style.css

```diff
@@ -336,7 +343,7 @@ input {
   margin-bottom: 8px;
 }
 
-textarea {
+/* textarea {
```

Comment:
- [ ] Let's remove commented code from your project/ This makes code hard to read.